baseframe/templates/baseframe/components.html

```diff
@@ -163,204 +163,38 @@
-      width: 185px;
-      padding: 22px 0px 0px 20px;
-      height: auto;
-      overflow-y: auto;
-      margin: 0;
-    }
-
-    .block-dropdown>li {
-      float: left;
-      height: auto;
-      list-style-type: none;
-      display: inline-block;
-      margin-right: 30px;
-      margin-bottom: 10px;
-      width: 3em;
-    }
-
-    .block-dropdown>li>a{
-      font-size: 12px;
-      text-align: center;
-      white-space: normal;
-      padding: 0;
-    }
-
-    .block-dropdown>li>a:hover{
-      background-color: transparent;
-      /*border: 1px solid #f2f2f2;*/
-    }
-
-    .block-dropdown>li>a>img{
-      max-width: 50px;
-      height: 30px;
-    }
-
-    a#hg-login-btn {
-      margin-top: 6px;
-      padding: 4px 10px;
-      margin-right: 4px;
-      margin-left: 12px;
-    }
-
-    a#hg-user-btn {
-      color:#777;
-    }
-
-    a#hg-login-btn:hover {
-      background-color: #285E8E;
-    }
-
-    ul.hg-header-app-nav {
-      margin-left: 6px;
-      margin-bottom: 0;
-    }
-
-    ul.hg-header-app-nav > li {
-      padding-left: 0;
-      padding-right: 0;
-      margin-right: 4px;
-    }
-
-    li.site-nav-collapse {
-      margin: 0 0 0 8px;
-    }
-
-    @media (min-width: 768px) {
-      ul.hg-header-app-nav {
-        margin-left: 0;
-      }
-
-      ul.hg-header-app-nav > li {
-        margin-right: -10px;
-      }
-    }
-
-    a.site-header > h1 {
-      font-size: 18px;
-      margin-bottom: -10px;
-      margin-top: 2px;
-    }
-
-    a.site-header > small {
-      font-size: 70%;
-    }
-
-    .hg-site-nav-toggle {
-      padding: 0 8px;
-      border: none;
-      margin-right: 2px;
-    }
-
-    a#hg-app-drawer {
-      color: #777;
-    }
-    
-    ul.hg-subnav {
-      margin-bottom: 0;
-    }
-
-    ul.hg-subnav > li > a {
-      padding: 12px 15px;
-    }
-  </style>
-  
+{% macro hgnav(site_title="HasGeek", site_nav=none, auth=true, network=true, links=[]) %}
   <nav class="navbar navbar-default navbar-fixed-top" id="hgnav" role="navigation">
     <div class="container">
-      <div class="navbar-header pull-left">
-        <ul class="list-inline hg-header-app-nav">
-          <li class="navbar-text pull-left dropdown site-nav-collapse">
-            {% if site_nav %}
-              <button type="button" data-toggle="collapse" data-target=".navbar-collapse" class="navbar-toggle pull-right hg-site-nav-toggle">
-                <i class="fa fa-bars"></i>
-              </button>
-            {% endif %}
-          </li>
-          <li>
+      <div class="navbar-header pull-left hg-header-app-nav">
             {{site_title}}
-          </li>
-        </ul>
       </div>
-      <div class="navbar-header pull-right">
+      <div class="navbar-header pull-right hg-header-nav">
         <ul class="nav pull-right">
           <li class="dropdown pull-left">
-            {% if hasgeek_auth %}
-              {{add_hasgeek_auth()}}
+            {% if auth %}
+              {{add_hasgeek_auth(links)}}
```

Comment:
As per our coding convention, Jinja2 tags should have a single space padding inside, so this should be `{{ add_hasgeek_auth(links) }}`.